theme/templates/resource-landing-page/manage-access.html

```diff
@@ -0,0 +1,242 @@
+                            </li>
+                            <li>
+                                <p><strong>Editors</strong> can edit resource metadata and add and delete resource
+                                    content files. If the resource
+                                    is Shareable editors can use manage access to extend editing or viewing access
+                                    to other users.</p>
+                            </li>
+                            <li><p><strong>Viewers</strong> can view resource metadata and download resource content
+                                files. If the resource is
+                                Shareable viewers can use manage access to extend viewing access to other users.</p>
+                            </li>
+                        </ul>
+                    </div>
+                    <div data-toggle="collapse" data-target="#usage-info"
+                       class="btn-show-more btn btn-default btn-xs">Show More</div>
+                    <br>
+                </div>
+
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">Who has access</h3>
+                    </div>
+                    <div class="panel-body panel-access ">
+                        <table class="table access-table" :style="{'pointer-events': isProcessing ? 'none' : 'auto' }">
+                            <tbody>
+                                {% include "includes/access-row.html" %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">Give access</h3>
+                    </div>
+                    <div id="div-invite-people" class="panel-body">
+                        <div id="invite-flag" class="btn-group space-bottom" role="group"
+                             aria-label="Share with users or groups">
+                            <button type="button"
+                                    :class="{btn: true, 'btn-primary': isInviteUsers, 'btn-default': !isInviteUsers}"
+                                    @click="isInviteUsers = true"
+                                    data-value="users">Users</button>
+                            <button type="button"
+                                    :class="{btn: true, 'btn-primary': !isInviteUsers, 'btn-default': isInviteUsers}"
+                                    @click="isInviteUsers = false"
+                                    data-value="groups">Groups</button>
+                        </div>
+
+                        <form id="add-access-form"
+                              :action="'/hsapi/_internal/' + resShortId + '/share-resource-with-' + (isInviteUsers ? 'user' : 'group') + '/'"
```

Comment:
Is there a way to have the JavaScript make the post, or will we always have a mix of forms/html POST and JS/Vue/jQuery POSTS? Just thinking big picture here.

Reply:
Consistency. I'm guessing we'll always have a mix.

Reply:
Actually, this form is not needed so I will be removing it. This post is now handled inside the Vue instance. I just forgot to remove it after moving the logic.

Reply:
I think we'll still have a few forms in place, mainly for actions that redirect, thus not needing to be Vue-instanciated. For example, creating a new version of a resource, deleting a resource, etc.